test/framework/compile.json

```diff
@@ -41,7 +41,8 @@
       "title": "Qooxdoo TestTAPper",
       "environment": {
         "qx.icontheme": "Tango",
-        "qxl.testtapper.testNameSpace": "qx.test"
+        "qxl.testtapper.testNameSpace": "qx.test",
+        "qx.debug": false
```

Comment:
As discussed on gitter, this looks like it will cause tests that require `qx.debug=true` to fail or not be tested properly, e.g., tests of `check` in properties. Do we really want to make this change?

Reply:
I suggest this steps

1. Accept this and setup a test pipline
2. Change compiler to optionally check settings at runtime
3. Revert this and test again

Reply:
Wouldn't it maybe be better to just skip this test instead until 2) is implemented, rather than changing qx.debug für all tests?

Reply:
We can do so. But qx.debug seems to have no effect on testing. After switching off no other test fails

Reply:
And there are some more tests which should be reenabled. I mean the php based remote test where the backend should be changed to an express based node server in compile.js

Reply:
I would see it this way: this test is testing something that is currently impossible because the compiler works differently from the generator. So the test itself makes no sense. The conclusion, to me, is that the test should be disabled rather than changing the environment to make the test think it succeeds when it really doesn't 😄

Reply:
> We can do so. But qx.debug seems to have no effect on testing. After switching off no other test fails

That seems to imply that we don't test that `check` works correctly, if nothing else. Ok, in the interest of getting tests working, I agree, we just go with this.